cypress/e2e/scenario-8-instance-composer.cy.js

```diff
@@ -141,759 +142,681 @@ if (Cypress.env("edition") === "iso") {
+      cy.get("button").contains("Edit").should("be.disabled");
+
+      //edit some of core attributes
+      cy.get('[data-type="app.ServiceEntityBlock"]')
+        .contains("many-defaults")
+        .click();
+      cy.get("button").contains("Edit").should("be.enabled");
+      cy.get("button").contains("Edit").click();
+
+      cy.get('[aria-label="TextInput-default_string"]').type(
+        "{selectAll}{backspace}updated_string",
+      );
+      cy.get('[aria-label="TextInput-default_int"]').type("{backspace}2");
+      cy.get('[aria-label="TextInput-default_float"]').type("{backspace}2");
+      cy.get('[aria-label="BooleanToggleInput-default_bool"]').within(() => {
+        cy.get(".pf-v5-c-switch").click();
+      });
+      cy.get('[aria-label="TextInput-default_empty_dict"]').type(
+        '{selectAll}{backspace}{{}"test2":"value2"{}}',
+      );
+      cy.get("button").contains("Save").click();
+
+      //edit some of embedded attributes
+      cy.get('[data-type="app.ServiceEntityBlock"]')
+        .contains("embedded")
+        .click();
+      cy.get("button").contains("Remove").should("be.disabled");
+      cy.get("button").contains("Edit").should("be.enabled");
+      cy.get("button").contains("Edit").click();
+
+      cy.get('[aria-label="TextInput-default_string"]').type(
+        "{selectAll}{backspace}updated_string",
+      );
+      cy.get('[aria-label="TextInput-default_int"]').type("{backspace}2");
+      cy.get('[aria-label="TextInput-default_float"]').type("{backspace}2");
+      cy.get('[aria-label="BooleanToggleInput-default_bool"]').within(() => {
+        cy.get(".pf-v5-c-switch").click();
+      });
+      cy.get('[aria-label="TextInput-default_empty_dict"]').type(
+        '{selectAll}{backspace}{{}"test2":"value2"{}}',
+      );
+      cy.get("button").contains("Save").click();
+
+      //remove extra_embedded instance
+      cy.get('[data-type="app.ServiceEntityBlock"]')
+        .contains("extra_embedded")
+        .click();
+      cy.get("button").contains("Remove").should("be.enabled");
+      cy.get("button").contains("Remove").click();
+
+      cy.get("button").contains("Deploy").click();
+
+      //assert that many-defaults is deployed and up
+      cy.get('[aria-label="InstanceRow-Intro"]').should("have.length", 1);
+      // await until parent_service is deployed and up
+      cy.get('[data-label="State"]', { timeout: 90000 }).should(
+        "have.text",
+        "up",
+      );
+
+      cy.get('[aria-label="ServiceInventory-Success"]').should("to.be.visible");
+      //assert that many-defaults is deployed and up
+      cy.get('[aria-label="InstanceRow-Intro"]').should("have.length", 1);
+      // await until parent_service is deployed and up
+      cy.get('[data-label="State"]', { timeout: 90000 }).should(
+        "have.text",
+        "up",
+      );
+
+      //check if core attributes and embedded entities are updated
+      cy.get("#expand-toggle0").click();
```

Comment:
You're checking in the collapsible rows, which will be gone. It would have been better to check in the new view.